sklearn/ensemble/forest.py

```diff
@@ -1631,6 +1631,19 @@ class labels (multi-output problem).
     reduce memory consumption, the complexity and size of the trees should be
     controlled by setting those parameter values.
 
+    Examples
+    --------
+    >>> from sklearn.ensemble import ExtraTreesClassifier
+    >>> from sklearn.datasets import make_classification
+    >>> X, y = make_classification(n_features=4, random_state=0)
+    >>> clf = ExtraTreesClassifier(n_estimators=100, random_state=0)
+    >>> clf.fit(X, y)
+    ExtraTreesClassifier(random_state=0)
+    >>> clf.feature_importances_
```

Comment:
I just noticed this it the inverse of my comment. We do not need this anymore because: https://github.com/scikit-learn/scikit-learn/blob/5271695cec52b519e1b3195223959472823127cc/setup.cfg#L7

Reply:
Oh I see. Didn't realize we have that as a default. My bad.

Reply:
yes please.